tpot/operator_utils.py

```diff
@@ -243,7 +243,7 @@ def export(cls, *args):
                 # To make sure the inital operators is the first parameter just
                 # for better persentation
                 for dep_op_pname, dep_op_str in dep_op_list.items():
-                    if dep_op_str == 'f_classif':
+                    if dep_op_str in ['f_classif', 'f_regression']:
```

Comment:
Check this [commit](https://github.com/rhiever/tpot/pull/538/commits/ea3b1945847f2ce5498c456a1a98c6dcdd4b2e3f) that is a more general way since `score_func` in scikit-learn only gets callable function.